pymobiledevice3/services/web_protocol/inspector_session.py

```diff
@@ -97,3 +121,71 @@ async def receive_message(self, message_id: int) -> Mapping:
                 return response
 
             self._responses_cache[receive_message_id] = response
+
+    # -- PARSE OBJECTS
+    def _parse_target_dispatch_message(self, response: Mapping):
+        print(response)
```

Comment:
Remove

Reply:
im not seeing it right now, think i already removed it